PaddleNLP/dialogue_system/dialogue_general_understanding/run.sh

```diff
@@ -3,7 +3,7 @@
 export FLAGS_sync_nccl_allreduce=0
 export FLAGS_eager_delete_tensor_gb=1
 
-export CUDA_VISIBLE_DEVICES=0
+export CUDA_VISIBLE_DEVICES=1
```

Comment:
setting cuda_visible_devices=1 may cause some errors when other people are using dgu, please change it to 0.